community/kernel/src/main/java/org/neo4j/kernel/impl/store/format/StoreVersion.java

```diff
@@ -24,39 +24,61 @@
 /**
  * All known store formats are collected here.
  */
-public class StoreVersions
+public enum StoreVersion
 {
-    // Community formats
-    public static final String STANDARD_V2_0 = "v0.A.1";
-    public static final String STANDARD_V2_1 = "v0.A.3";
-    public static final String STANDARD_V2_2 = "v0.A.5";
-    public static final String STANDARD_V2_3 = "v0.A.6";
-    public static final String STANDARD_V3_0 = "v0.A.7";
+    STANDARD_V2_0( "v0.A.1", true ),
+    STANDARD_V2_1( "v0.A.3", true ),
+    STANDARD_V2_2( "v0.A.5", true ),
+    STANDARD_V2_3( "v0.A.6", true ),
+    STANDARD_V3_0( "v0.A.7", true ),
 
-    // Enterprise formats
-    public static final String HIGH_LIMIT_V3_0 = "vE.H.0";
+    HIGH_LIMIT_V3_0( "vE.H.0", false );
+
+    private static final StoreVersion[] ALL_STORE_VERSIONS = values();
+
+    private final String string;
+    private final boolean isCommunity;
+
+    StoreVersion( String string, boolean isCommunity )
+    {
+        this.string = string;
+        this.isCommunity = isCommunity;
+    }
+
+    public String string()
```

Comment:
Can we name it to something else then `string`? for example `storeVersion()` since all the usages assign to variable with that name